.github/workflows/ci.yml

```diff
@@ -272,7 +272,6 @@ jobs:
         ceci --dry-run examples/2.2i/pipeline.yml
         ceci --dry-run examples/buzzard/pipeline.yml
         ceci --dry-run examples/cosmodc2/pipeline.yml
-        ceci --dry-run examples/dr1b/pipeline.yml
```

Comment:
why did this get dropped?

Reply:
It was just so old at this point, and superseded by 2.2i.